### PR TITLE
add grid strat

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes ready for review (this is noarch)

File: recipes/grid-strategy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib
```
Comment (beckermr) [inline recipes/grid-strategy/meta.yaml:23]: matplotlib-base

Comment (CJ-Wright) [reply recipes/grid-strategy/meta.yaml:23]: Ahh, this was created by grayskull, I'll also add an issue upstream.

Comment (CJ-Wright) [inline recipes/grid-strategy/meta.yaml:23]:
```suggestion
    - matplotlib-base
```

**Comment by CJ-Wright** [comment]
@beckermr ready for another round

**Comment by CJ-Wright** [comment]
Thank you!

**Pull request merged**